com.gitlab.davem.ClamTk.yaml

```diff
@@ -1,18 +1,15 @@
 app-id: com.gitlab.davem.ClamTk
 runtime: org.freedesktop.Platform
-runtime-version: '20.08'
+runtime-version: '22.08'
 sdk: org.freedesktop.Sdk
 command: clamtk
-rename-icon: clamtk
```

Comment:
Why remove these lines? They should have removed the need for the manual icon duplicate & desktop file edits below.

Reply:
Yea I replaced them with `install`, when appdata validation was failing. I thought it was causing some issue, will restore.